kinesis_manager/src/stream_definition_provider.cpp

```diff
@@ -33,7 +35,7 @@ KinesisManagerStatus StreamDefinitionProvider::GetCodecPrivateData(
     return KINESIS_MANAGER_STATUS_INVALID_INPUT;
   }
   std::string b64_encoded_codec_private_data;
-  reader.ReadStdString(prefix + "codecPrivateData", b64_encoded_codec_private_data);
+  reader.ReadParam(prefix + "codecPrivateData", b64_encoded_codec_private_data);
```

Comment:
Will this still work with the explicit constructor for parameter path?

Reply:
Yep, `prefix` is an already constructed ParameterPath object.